.travis.yml

```
@@ -21,11 +21,11 @@ env:
      - WORKSPACE="ReactiveLists.xcworkspace"
      - FRAMEWORK_SCHEME="ReactiveLists"
      - EXAMPLE_SCHEME="Example"
-     - SDK=iphonesimulator12.1
+     - SDK=iphonesimulator12.2
```

Just do this, and it'll use the latest by default
```suggestion
     - SDK=iphonesimulator
```